js/marquee.js

```diff
@@ -1,12 +1,18 @@
 //marquee.js
 
-	var assocWrap = document.getElementById('assoc_slide_wrap');
-	var pos = -210;
-	var moveMarquee = setInterval ("marquee()", 4000);
+	// Here's another way to declare multiple vars:
```

Comment:
This is just a style preference. I like combining like variables into one var call like this. It gives other developers, not to mention my future self, clues about how I think.